tests/acceptance/acceptance/forms/FormGetErrors.feature

```diff
@@ -0,0 +1,54 @@
+@symfony-common
+Feature: Form getErrors return type provider
+
+  Background:
+    Given I have Symfony plugin enabled
+  Scenario: getErrors test
+    Given I have the following code
+          """
+      <?php
+
+      use Symfony\Component\Form\FormInterface;
+
+      function foo(FormInterface $form): void
+      {
+          foreach ($form->getErrors() as $error1) {
+              /** @psalm-trace $error1 */
+          }
+
+          foreach ($form->getErrors(true) as $error2) {
+              /** @psalm-trace $error2 */
+          }
+
+          foreach ($form->getErrors(false) as $error3) {
+              /** @psalm-trace $error3 */
+          }
+
+          foreach ($form->getErrors(true, true) as $error4) {
+              /** @psalm-trace $error4 */
+          }
+
+          foreach ($form->getErrors(true, false) as $error5) {
+              /** @psalm-trace $error5 */
+          }
+
+          foreach ($form->getErrors(false, true) as $error6) {
+              /** @psalm-trace $error6 */
+          }
+
+          foreach ($form->getErrors(true, false) as $error7) {
```

Comment:
looks good to me but I don't understand how error5 (L:31) is different than this one?

Reply:
Nice catch ; seems like this is not working and the test is wrong.

Every error are still Symfony\Component\Form\FormError\|Symfony\Component\Form\FormErrorIterator
and the test is only checking it contains `Symfony\Component\Form\FormError`.

I'll update the test